src/material/list/list-option.scss

```diff
@@ -1,17 +1,22 @@
 @use 'sass:map';
-@use '@angular/cdk';
 @use '@material/checkbox/checkbox' as mdc-checkbox;
-@use '@material/list/evolution-variables' as mdc-list-variables;
 @use '@material/checkbox/checkbox-theme' as mdc-checkbox-theme;
 
 @use '../core/mdc-helpers/mdc-helpers';
 @use '../checkbox/checkbox-private';
 @use './list-option-trailing-avatar-compat';
+@use './list-item-hcm-indicator';
 
 // For compatibility with the non-MDC list, we support avatars that are shown at the end
 // of the list option. We create a class similar to MDC's `--trailing-icon` one.
 @include list-option-trailing-avatar-compat.core-styles($query: mdc-helpers.$mdc-base-styles-query);
 
+.mat-mdc-list-option.mdc-list-item--selected {
```

Comment:
FYI: It might be ok to move the styles to the top like this, but its safer to leave them in the same place, since order is the tie breaker for specificity

Reply:
good catch, I'll leave the styles in the same place